README.md

```diff
@@ -35,4 +35,42 @@ The following templates will override the out-of-the-box theme templates.
 - featured products
 - image_text_overlay
 
-**Compatible with StoreConnect v16.0.0 and on**
\ No newline at end of file
+### Setting up the Header
+
+**Top header**
+
+This is the white section at the top of your header and it features a small slideshow on the left for short content, with two customizable links on the right.
+
+If this component is not needed, you can easily deactivate it by setting the `active.top.header` theme variable to `inactive`, `off`, or any other string.
```

Comment:
Check this is true:

```suggestion
If this component is not needed, you can easily deactivate it by setting the `active.top.header` theme variable to `inactive`, `off`, or any string other than `active`.
```

Reply:
@toddweetman Yes, I have verified it and it works.